### PR TITLE
Add slideshare

### PR DESCRIPTION
**Pull request by MarquiseRosier**
related to #735

### CODE REVIEW

File: docs/action-properties-secrets.md [hunk 1]
```diff
@@ -113,7 +113,7 @@ Comma-separated list of allowed hostnames for embeds. Supports `*.example.com` a
 The default value is:
 
 ```json
-"www.youtube.com, spark.adobe.com, unsplash.com, soundcloud.com, lottiefiles.com, www.slideshare.net, vimeo.com, www.instagram.com, twitter.com, open.spotify.com, web.spotify.com, player.vimeo.com, www.linkedin.com, w.soundcloud.com"
+"www.youtube.com, spark.adobe.com, unsplash.com, soundcloud.com, lottiefiles.com, www.slideshare.net, vimeo.com, www.instagram.com, twitter.com, open.spotify.com, web.spotify.com, player.vimeo.com, www.linkedin.com, w.soundcloud.com, www.slideshare.net"
```
Comment (rofe) [inline docs/action-properties-secrets.md:116]: @MarquiseRosier `www.slideshare.net` is now listed twice...

File: docs/secrets.md [hunk 2]
```diff
@@ -113,7 +113,7 @@ Comma-separated list of allowed hostnames for embeds. Supports `*.example.com` a
 The default value is:
 
 ```json
-"www.youtube.com, spark.adobe.com, unsplash.com, soundcloud.com, lottiefiles.com, www.slideshare.net, vimeo.com, www.instagram.com, twitter.com, open.spotify.com, web.spotify.com, player.vimeo.com, www.linkedin.com, w.soundcloud.com"
+"www.youtube.com, spark.adobe.com, unsplash.com, soundcloud.com, lottiefiles.com, www.slideshare.net, vimeo.com, www.instagram.com, twitter.com, open.spotify.com, web.spotify.com, player.vimeo.com, www.linkedin.com, w.soundcloud.com, www.slideshare.net"
```
Comment (rofe) [inline docs/secrets.md:116]: `www.slideshare.net` is now listed twice...

File: src/schemas/secrets.schema.json [hunk 3]
```diff
@@ -38,7 +38,7 @@
     "EMBED_WHITELIST": {
       "type": "string",
       "description": "Comma-separated list of allowed hostnames for embeds. Supports `*.example.com` as a subdomain wildcard. Use `*` to allow all embeds (potentially insecure and conflicting with `DATA_EMBED_WHITELIST`)",
-      "default": "www.youtube.com, spark.adobe.com, unsplash.com, soundcloud.com, lottiefiles.com, www.slideshare.net, vimeo.com, www.instagram.com, twitter.com, open.spotify.com, web.spotify.com, player.vimeo.com, www.linkedin.com, w.soundcloud.com"
+      "default": "www.youtube.com, spark.adobe.com, unsplash.com, soundcloud.com, lottiefiles.com, www.slideshare.net, vimeo.com, www.instagram.com, twitter.com, open.spotify.com, web.spotify.com, player.vimeo.com, www.linkedin.com, w.soundcloud.com, www.slideshare.net"
```
Comment (rofe) [inline src/schemas/secrets.schema.json:41]: `www.slideshare.net` is now listed twice...

Comment (MarquiseRosier) [reply src/schemas/secrets.schema.json:41]: well this is embarrassing, don't know how I completely missed that. I'll update this, I'm trying to figure out why the new embeds won't work, you can expect a PR soon that will correct this.

**Pull request merged**